src/plugins/intel_gna/descriptions/gna_flags.hpp

```diff
@@ -18,5 +18,6 @@ struct GNAFlags {
     bool sw_fp32 = false;
     bool performance_counting = false;
     bool input_low_precision = false;
+    bool enable_user_warning = false;
```

Comment:
I suppose it should be enum as well if we will decide to support other values besides `NONE` and `WARNING`

Reply:
As above.